manifests/10-dedicated-admin-operator.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-dedicated-admin
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true"
+---
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: dedicated-admin-operator
+  namespace: openshift-dedicated-admin
+spec:
+  targetNamespaces:
+  - openshift-dedicated-admin
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: dedicated-admin-operator
+  namespace: openshift-dedicated-admin
+spec:
+  channel: bundle-manifests
```

Comment:
I understand this will be re-generated every time, but shouldn't we push a version to the master branch having channel=master as well?

Reply:
I can manually update it in this PR to something bogus, forcing a regen.